src/lib/services/events-service.ts

```diff
@@ -53,6 +55,14 @@ export const getEndpointForRawHistory = ({
     },
     true,
   );
+
+  const url = new URL(route);
+
+  if (!tokens.accessToken) return url.toString();
+
+  url.searchParams.append('accessToken', tokens.accessToken);
```

Comment:
Does ui-server have the logic to use this search param? Wasn't seeing anything in there

Reply:
Sorry, I meant to put this back in draft. I am working on that side of things now.